doc/modules/cross_validation.rst

```diff
@@ -554,3 +554,19 @@ Cross validation and model selection
 Cross validation iterators can also be used to directly perform model
 selection using Grid Search for the optimal hyperparameters of the
 model. This is the topic of the next section: :ref:`grid_search`.
+
+Cross validation of time series data
+====================================
+
+Time series data is characterised by the correlation between observations 
+that are near in time (*autocorrelation*). However, the "classical" k-times 
```

Comment:
Firstly, it's k-fold, not "k-times". Secondly other classical approaches, such as `ShuffleSplit` and `LeaveOneLabelOut` have the same problem.